portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,6 +40,10 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  Translate translate = TranslateOptions.getDefaultInstance().getService();
+  Translation translation;
+  String translatedText;
```

Comment:
I think this can just be a local variable in doGet() in the for loop when you call getTranslatedText()